test/openapi-clients/dotnet/SrsClient/clientsdk/src/IO.Swagger/Client/ApiClient.cs

```diff
@@ -29,7 +29,8 @@ public partial class ApiClient
     {
         private JsonSerializerSettings serializerSettings = new JsonSerializerSettings
         {
-            ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor
+            ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
+            MaxDepth = int.MaxValue
```

Comment:
Is this code generated? Maybe it should be regenerated instead with a higher version of the generator?

Reply:
There is no new version of the generator, which to bump the newtonsoft json dependency to 13.0.1 and handle the serialization depth issue.